test/smoke/src/helpers/appiumHelper.ts

```diff
@@ -77,12 +103,112 @@ export class AppiumHelper {
         };
     }
 
+    public static prepareAttachOptsForIosApp(deviceName: string, appPath: string) {
+            return {
+                desiredCapabilities: {
+                    browserName: "",
+                    platformName: "iOS",
+                    platformVersion: this.getiOSPlatformVersion(),
```

Comment:
I like this idea, we can do the same for Android to get more flexibility.

Reply:
@SounD120  I'd do that in separate PR